x/leverage/spec/03_queries.md

```diff
@@ -11,6 +11,7 @@ Queries on accepted asset types:
 - **Borrow APY** queries for the [Borrow APY](01_concepts.md#Borrow-APY) of a specified denomination.
 - **Lend APY** queries for the [Lending APY](01_concepts.md#Lending-APY) of a specified denomination.
 - **Reserve Amount** queries for the amount reserved of a specified denomination.
+- **Total Collateral** queries for the total collateral amount of a specified uToken denomination.
```

Comment:
side comment, I'm not sure we need this doc. All queries are defined in proto, and we can generate documentation from proto.

Reply:
Do you know if there's a convenient way to get proto-generated docs into the spec folder?

I would like the spec to keep a list of queries and message types since they for the API, but autogenerated query docs in particular would be better than the current list.

Reply:
I'm pretty sure buf can do it. I will add a task for that.

Reply:
https://github.com/umee-network/umee/issues/1097